frontend/src/app/lightning/nodes-channels-map/nodes-channels-map.component.ts

```diff
@@ -206,20 +205,13 @@ export class NodesChannelsMap implements OnInit {
   prepareChartOptions(nodes, channels) {
     let title: object;
     if (channels.length === 0 && !this.placeholder) {
-      this.chartOptions = null;
-      this.showIndexingInProgress = true;
       this.isLoading = false;
-      return;
-    }
-
-    // empty map fallback
-    if (channels.length === 0 && this.placeholder) {
```

Comment:
I think this condition is for the node preview component used by unfurler so it looks your change would change its behavior. So need to make sure it does not break it.

Reply:
OK I believe I restored the preview "preview component" behavior